profiles/serializers.py

```diff
@@ -244,3 +252,49 @@ class Meta:
         model = User
         fields = ("id", "username", "profile", "email")
         read_only_fields = ("id", "username")
+
+
+class ProgramCertificateSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Program Certificates
+    """
+
+    class Meta:
+        model = ProgramCertificate
+        fields = "__all__"
+
+
+class ProgramLetterTemplateFieldSerializer(serializers.Serializer):
+    id = serializers.IntegerField()
+    meta = serializers.JSONField()
+    title = serializers.CharField()
+    program_id = serializers.IntegerField()
+    program_letter_footer = serializers.JSONField()
+    program_letter_footer_text = serializers.CharField()
+    program_letter_header_text = serializers.CharField()
+    program_letter_text = serializers.CharField()
+    program_letter_logo = serializers.JSONField()
+    program_letter_signatories = serializers.ListField(child=serializers.JSONField())
+
+
+class ProgramLetterSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Program Letters
+    """
+
+    id = serializers.UUIDField(read_only=True)
+
+    template_fields = serializers.SerializerMethodField()
+
+    certificate = ProgramCertificateSerializer()
+
+    @extend_schema_field(ProgramLetterTemplateFieldSerializer())
+    def get_template_fields(self, instance) -> dict:
+        """Return the number of episodes in the podcast"""
```

Comment:
Wrong docstring.
"Get the serialized template fields"?

Reply:
good catch!